### PR TITLE
Awesome books Project

### PR DESCRIPTION
**Pull request by Estherstarr**
We created the following requirements

- [ ] Create a collection that keeps a list of books.
- [ ] Create a function to add a new book to the collection, with title and author.
- [ ] Create a function to remove a book from the collection 
- [ ] Display all books saved in the collection in the top part of the page.
- [ ] Preserve data in the browser's memory by using Local Storage

Screenshot 2022-03-22 at 8 15 48 AM
Screenshot 2022-03-22 at 8 15 27 AM
Screenshot 2022-03-22 at 8 13 33 AM

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  const libraryContainer = document.createElement('div');
+
+  bookList.forEach((book) => {
+    libraryContainer.appendChild(displayBook(book));
+    currentBooks.appendChild(libraryContainer);
+  });
+};
+
+window.onload = displayBooks;
+
```
Comment (BenMukebo) [inline app.js]: - [ ] You are calling the `displaybooks` function on window load, however, the book should be displayed when we click on the add button, please fix that.
second

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@Estherstarr,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight 💪🏻

- No linter errors. ✔️ 
- Good commit messages ✔️ 
- Professional README ✔️ 
- You have followed  es6 best practices. ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Estherstarr** [comment]
Required change
- [ ] Display booklist when the add button is clicked

File: index.html [hunk 2]
```diff
@@ -1,12 +1,46 @@
...
+              placeholder="Author"
+            />
+            <small></small>
+          </div>
+          <button type="submit" class="btn" id="add-book">Add</button>
+        </form>
+      </section>
+    </main>
+
+    <script src="app.js"></script>
```
Comment (AnselemOdims) [inline index.html:44]: - [ ] [OPTIONAL] I am not sure what code is adding the below image to the user interface, but it does not give a good user experience.Kindly remove it

image

**Review by AnselemOdims — APPROVED** [review]
### Hi  🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Estherstarr) [reply index.html:44]: Thank you @AnselemOdims

**Pull request merged**